pyomo/contrib/pynumero/interfaces/tests/_external_grey_box_models.py

```diff
@@ -0,0 +1,546 @@
+from pyomo.contrib.pynumero.dependencies import (
```

Comment:
Why does this file name begin with an underscore?

Reply:
Just to indicate that it was something private to be used by the tests. I will change it.

Reply:
done.